src/containers/form/FormContainer.js

```diff
@@ -101,11 +101,13 @@ class Form extends React.Component<Props, State> {
     validateOnInput(name, input, fieldType, formatErrors, setErrorsFn);
   }
 
-  handleDateInput = (e, section, name) => {
-    const input = e;
+  handleDateInput = (e, section, name, formatErrors, setErrorsFn) => {
+    let input = e;
+    input = input.replace(/T(.*)$/g, 'T00:00:00.000Z');
```

Comment:
This is hacky, but we're stuck dealing with a bad dependency.

However, I believe all of the properties that are `Date` only care about the day+month+year.

Can you try just removing everything after and including the `T`, basically submitting only `2017-11-20`, and seeing if that works?

Reply:
Fixed. The shorter date submits fine.